test/Extensions/TesterAdoNet/RelationalUtilities/PostgreSqlStorageForTesting.cs

```diff
@@ -8,6 +8,8 @@ namespace Tester.RelationalUtilities
 {
     class PostgreSqlStorageForTesting : RelationalStorageForTesting
     {
+        protected override string ProviderMoniker => "PostgreSQL";
```

Comment:
Is this the same as AdoNetInvariant?

Reply:
No, those are the full names of the drivers to use.